lib/orch_zmq_config.cpp

```diff
@@ -35,12 +35,20 @@ std::set<std::string> swss::load_zmq_tables()
 int swss::get_zmq_port()
 {
     auto zmq_port = ORCH_ZMQ_PORT;
-    if (const char* nsid = std::getenv("NAMESPACE_ID"))
+    const char* nsid = std::getenv("NAMESPACE_ID");
```

Comment:
`string nsid_str = std::getenv("NAMESPACE_ID");`
Line 39 seems redundant

Reply:
It could be a null pointer, so risky to convert implicitly (even though the syntax works)

Reply:
Ok, I see. Can we do it in the following way? I really don't like that we check nsid twice
```
   const char* nsid = std::getenv("NAMESPACE_ID");
    std::string nsid_str = nsid ? std::string(nsid) : "";
    if (nsid != nullptr)
    {
        try
        {
            // namespace start from 0, using original ZMQ port for global namespace
            zmq_port += std::stoi(std::str(nsid)) + 1;
        }
        catch (...)
        {
            SWSS_LOG_ERROR("Failed to convert %s to int, fallback to default port", nsid_str.c_str());
        }
```

Reply:
Resolved offline